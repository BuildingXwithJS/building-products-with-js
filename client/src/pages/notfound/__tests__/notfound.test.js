// our packages
import NotFoundPage from '../index';

test('# Not found page', () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
