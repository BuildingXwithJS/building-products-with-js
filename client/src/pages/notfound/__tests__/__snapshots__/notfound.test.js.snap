exports[`test # Not found page 1`] = `
<h1>
  Page not found
</h1>
`;
